src/input.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-template-curly-in-string */
+const yup = require("yup");
+const { PopulationPresets } = require("./presets");
+const { QuarantineTypes } = require("./disease/quarantine");
+
+const InputSchema = yup.object().shape({
+    populationSize: yup.number()
+        .positive()
```

Comment:
Don't know if this is needed for some reason, but should be redundant with the min(9) restriction, right?